backend/dashboardd/dashboardd.go

```diff
@@ -221,8 +220,6 @@ func newBackendProxy(host string, port int, TLS *types.TLSOptions) (*httputil.Re
 
 	// Configure TLS
 	if TLS != nil {
-		target.Scheme = "https"
-
```

Comment:
I wonder if we can remove `cfg.InsecureSkipVerify = true` from L227 now? I think this is an artifact from when the dashboard statically defined `localhost` as the APIUrl.

https://github.com/sensu/sensu-go/blob/add72d2f5c001c67ff94ce440b3d642a479795dc/backend/dashboardd/dashboardd.go#L227

Reply:
We are now using it EVERYWHERE instead 🤣

Reply:
not sure if I want to laugh or cry